### DISCUSSION TITLE
Stella

### DISCUSSION THREAD

**Discussion by Aqueuse** [Aqueuse/NekoV2#5] [Announcements]
A fully public version, with an installer and a settings page to add your own assets and configuration :) 

<hr /><em>This discussion was created from the release <a href='https://github.com/Aqueuse/NekoV2/releases/tag/'>Stella</a>.</em>

**Comment by Aqueuse** [comment]
Bugs, problèmes ? Dits moi tout !  N'oubliez pas de me préciser votre système d'exploitation ;)

**Reply by miyucode** [reply]
ça me dis qu'il a un virus :')

**Comment by Aqueuse** [comment]
salut, c'est normal si tu as un message de windows, c'est un message qui s'affiche parce qu'il faudrait que  j'enregistre l'application mais ça coute de l'argent, donc l'editeur est set sur inconnu à la place. Mais pas de panique, tout va très bien. Mon chaton est garanti sans bug ni puces =^^=
